azure-pipelines.yml

```diff
@@ -2,27 +2,28 @@ trigger:
   branches:
     include:
     - main
+
   paths:
     include:
     - azure-pipelines.yml
     - 'Fakebook.Profile'
 
 pr:
-- 'main'
+- '*'
```

Comment:
path trigger for ci builds is limited but for pr builds it's not. they should be consistent.